<!DOCTYPE html>
<html lang="en">

<head>
    <meta charset="utf-8">

    <title>Notitia | Our Science</title>
    <meta name="description" content="Homepage for NotitiaBio">
    <meta name="author" content="Notitia">
    <meta name="viewport" content="width=device-width, initial-scale=1">


    <link rel="stylesheet" href="style.css">
    <link rel="icon" href="images/small-logo.png" type="image/png">

    <link rel="stylesheet" href="https://unpkg.com/aos@next/dist/aos.css" />
    <script src="https://kit.fontawesome.com/5d869ca34c.js" crossorigin="anonymous"></script>
</head>
<main>
    <div class="hamburger">
        <div></div>
        <div></div>
        <div></div>
    </div>
    <nav class="top-nav">
        <img src="images/Original.svg" alt="notitia logo">
        <a href="index.html">Home</a>
        <a href="about.html">About Us</a>
        <a href="science.html" class="active">Our Science</a>
        <a href="general-public.html">General Public</a>
        <a href="patients.html">Patients</a>
        <a href="investors.html">Investors & News</a>
    </nav>
    <button class="contact-us" id="contact-us">Contact Us</button>
    <section class="hero science-hero">
        <div class="science-text">
            <h1>Our Science</h1>
        </div>
    </section>

    <section class="frame grid banner-2" data-aos="shrink">
        <h1>The Four Pillars of Understanding Microbiome</h1>
        <p>The complex and dynamic nature of gut microbiota raises significant challenges for understanding its role in human health and diseases. To overcome these challenges, our scientific co-founder, Dr. Liping Zhao, has spent the past three decades to develop a conceptual framework encompassing Four Pillars to guide microbiome research and promote understanding of the role of gut microbiome in human health and diseases.</p>
    </section>

    <section class="frame " data-aos="shrink">
        <div>
            <h1>Pillar 1: We must move from studying association to studying causation.</h1>
            <p>
                To date, plenty studies have shown that patients of various diseases have different microbiota compared to healthy controls. There is also significant evidence identifying diet as the main force for shaping the composition of the gut microbiota. Thus, the key to changing the gut microbiota lies in dietary changes. However, it is still unclear whether changes in a patient’s gut microbiota is the cause or the end result of disease occurrence? Changing one’s diet can simultaneously impact gut microbiota composition and patient health outcome. However, does the changed gut microbiota also improve health? Or is it merely an end result of the improved diet? 
                <br><br>
                Some bacteria are more frequently found in healthy people, while other bacteria are more frequently found in patients. Nevertheless, such association evidence can not prove whether a bacterium is beneficial or harmful. Moving from studying association to studying causality, we must isolate the gut bacteria and inoculate them into germ-free animal models to test their health effects. In 2012, Dr. Zhao foun a particular pathogen overgrown in a morbidly obese patient's gut. After receiving a microbiome-targeted diet, the pathogen quickly reduced from 1/3 of the patient’s gut bacteria population to a non-detectable level. At the same time, this patient lost 110 lbs of his initial body weight and recovered from Type II Diabetes. Dr. Zhao hypothesized that this pathogen played a causal role in the patient's obesity and Type II Diabetes. This pathogen was then isolated from the patient's baseline samples and transferred to a germ-free mice gut. Lab results clearly showed that THIS pathogen induced obesity, fatty liver, and early diabetes in the recipient mice. Dr. Zhao then identified the exact molecule from the pathogen bacteria that is responsible for causing the diseases. For the first time,  science proved that certain “bad” gut bacteria can indeed CAUSE obesity and Type II Diabetes.
                <br><br>
                In 2018, Dr. Zhao found that some bacteria are scarce in the gut of Type II Diabetes patients. After receiving a microbiome-targeted diet, these bacteria became abundant in the gut. Meanwhile, patients showed significant improvement in their health indicators. To prove causality, Dr. Zhao transferred the gut microbiota from these patients to germ-free mice. He found that the recipient mice developed diabetes. He then isolated the potentially beneficial bacteria and transferred them to the diabetic mice. Amazingly, the bacteria helped mice recover from their diabetes. These bacteria are indeed beneficial!
                <br><br>
                We believe that conducting causality studies with a molecular level of rigor is key to establish our confidence to use bacteria and their effector molecules as biomarkers and targets for disease prevention or treatment.
            </p>
        </div>
    </section>

    <section class="frame step-1" data-aos="shrink">
        <div>
            <h1>Pillar 2: We must identify “good” and “bad” bacteria at the strain level. </h1>
            <p>
                Strains are the undividable living units of bacteria. A useful analogy: One strain = One person.
                <br><br>
                As a species, human individuals are almost genetically identical, with only 0.1% genetic variance between each other. Therefore, humans are biologically similar to each other. However, bacterial strains belonging to the same species can have up to 30% genetic variance. In contrast, humans and mice only share a 10% genomic difference.
                <br><br>
                Because of such vast genetic differences, some bacteria can be pathogenic and others can be beneficial within the same bacterial species. We believe that strain-level analysis is key to identifying whether a bacterium is good or bad to health. We cannot collectively talk about bacterial species as good or bad. For example, Dr. Zhao found one strain that can aggravate gut inflammation and colitis in a supposedly beneficial species of gut bacteria. In another example, he discovered that a healthy diet does not promote all of the strains belonging to a supposedly beneficial species. Many of the strains were, in fact, non-responders to the healthy diet.
                <br><br>
                We cannot use groups such as species or genus as functional units to understand their role in human health and diseases because their members often do not function in the same way. We need to do everything we can to go down to the strain level if we desire to identify vitally important gut microbiota members to human health and diseases.
            </p>
        </div>
    </section>

    <section class="frame step-2" data-aos="shrink">
        <div>
            <h1>Pillar 3: We must remember that bacteria form ecological guilds.</h1>
            <p>
                Members in a macro-ecosystem (e.g., a rain forest) are not independent of each other. Rather, they form an ecological structure called guild: each guild is a coherent functional group of bacterial members from various taxonomic positions. Members of the same guild thrive or decline together and consequently they show co-abundant behavior. Dr. Zhao found that bacterial strains in the gut also organize themselves into guilds. In 2015, Dr. Zhao identified three beneficial and nine harmful guilds in the gut of genetically obese children. This was the first time that the guild concept was utilized in human microbiome research. 
                <br><br>
                In macro ecology, we understand that not all members within an eco-system are equally important. The tall tree species in the rain forest are known as the foundation species because they structure and stabilize the whole ecosystem. Foundation species are different from keystone species. If you lose a keystone species, you may lose part of the ecosystem. If you lose the foundation species, you will eventually lose everything. Are there any gut bacteria that also work as the foundation species to structure and stabilize the healthy gut microbiota?
                <br><br>
                In 2018, Dr. Zhao discovered a guild of beneficial bacteria. When promoted to a high level, this guild of beneficial bacteria can effectively change the entire gut environment and make it unfavorable to pathogens in the gut. This guild plays a fundamentally important role in structuring and stabilizing the healthy gut microbiota. Bacteria from 3 different phyla can be found in this guild. We have named this group of bacteria the "Foundation Guild" of the gut microbiome.                 <br><br>
                Some Foundation Guild members seem to be shared by everybody, and others may be specific to the individual, family, or ethnic groups. Different members of the Foundation Guild have distinct nutritional requirements. It is possible to design particular formulas to promote the Foundation Guild found in various individuals and ethnic groups.
            </p>
        </div>
    </section>

    <section class="frame step-3" data-aos="shrink">
        <div>
            <h1>Pillar 4: We must not rely on existing reference databases for microbiome research. </h1>
            <p>
                Our current knowledge of the gut microbiome is still limited. If we relied on existing reference databases to analyze microbiome data, we would limit research to what is already known, re-discover what others have previously found, and potentially over look novel bacteria that are important to human health.
                <br><br>
                We need to adopt a system-level, non-targeted, discovery approach for understanding gut microbiota in human health and diseases. We need to develop a reference-independent strategy for analyzing microbiome data. In the potentially beneficial guilds identified by Dr. Zhao, only about 20% can be given a species name.
            </p>
        </div>
    </section>

    <section class="frame" data-aos="shrink">
        <div>
            <h1>Notitia's Foundation GuildTM Microbiome R&D Platform</h1>
            <p>
                Based on these Four Pillars and the ground-breaking scientific discoveries, we have developed the Foundation GuildTM Microbiome R&D Platform that serves as the basis of various technologies for promoting the Foundation GuildTM in patients and the generally healthy population. 
                <br><br>
                Notitia's platform and technologies are centered on the Foundation GuildTM gut bacteria, which has been proven to have a causal relationship with human health and diseases. Our proprietary data analysis pipeline allows us to study members of the Foundation GuildTM at the strain-level and examine relationships within their respective guilds. This entire process does not rely on existing references, enabling us to discover any previously un-discovered bacteria that could be important to human health. 
                <br><br>
                One of the most important aspects of our platform is its ability to analyze and adjust individual Foundation GuildTM gut bacteria. The platform is made up of three unique technologies:
                
            </p>
            <ol>
                <li>Testing the Foundation GuildTM
                    <br><br>
                    Notitia has created a predictive analysis method to identify the Foundation GuildTM in each person's gut microbiota. We can analyze an individual's fecal sample and investigate: does this individual have Foundation GuildTM bacteria in his gut? If so, what are the specific members of his/her Foundation GuildTM? What kind of nutrients are required to support their growth?
                    <br><br>
                    The analysis can also predict the optimal state of the Foundation GuildTM for each person and keep track of their recovery and progression towards that optimal state. 
                    <br><br>
                    (Example graphics) In the example on the left, the analysis predicted the optimal healthy state of the Foundation GuildTM of the individual. After being on the microbiome nutrition, the same test showed that the individual's gut microbiome moved closer to its optimal state, indicating that the Foundation GuildTM is recovering. 
                    <br><br>
                </li>
                <li>Growing the Foundation GuildTM
                    <br><br>
                    Notitia has invented methods to isolate and culture the Foundation GuildTM bacteria from fecal samples as a consortium of defined membership. We can exclude all other bacteria and other micro-organisms inside the fecal sample and focus insolate the Foundation Guild bacteria. Regardless of the number of specific bacteria one person has in the Foundation GuildTM, our method allows us to isolate the entire group as a whole.
                    <br><br>
                    This technology enables us to store each person's health Foundation Guild bacteria in cryogenic storage for future uses or create LBP (live biotherapeutic products) drugs, which contain the Foundation GuildTM bacteria mass-produced to administer to patients that need microbiota transplantation. (example graphic)
                    <br><br>
                </li>
                <li>Nutrition for the Foundation GuildTM
                    <br><br>
                    Notitia has designed formulas for patients and the generally healthy that contains the right nutrients to support the Foundation GuildTM's growth in the gut. The formulas contain critical ingredients such as complex carbohydrates and phytochemicals with antioxidant activity. Each ingredient is designed to enhance specific Foundation GuildTM members and ensure maximum delivery and utilization by the Foundation GuildTM.
                    <br><br>
                    The patient version of this formula focuses more on supporting as many types of Foundation Guild bacteria in the human gut. In comparison, the generally healthy version focuses more on versatility in applications and aiding the most common Foundation Guild members found in every human gut.
  
                </li>
            </ol>
        </div>

    </section>


        <div class="comparison">
            <!-- <h1 class="comparison-header seaweed-text">Notitia</h1> -->
            <div class="comparison-header">
                <img src="images/notitia.png">
            </div>
            <h1 class="comparison-vs">VS.</h1>
            <h1 class="comparison-header gray-text">Competitor</h1>
            <div class="paragraph comparison-item seaweed-text">
                <h4>Guild-based Analysis</h4>
                <ul>
                    <li>Does not rely on existing database.</li>
                    <li>Able to discover novel bacteria in the microbiome.
                    </li>
                </ul>
                </div>
                
                <div class="paragraph comparison-item gray-text">
                <h4>Taxon-based Analysis
                </h4>
                <ul>
                    <li>Relies on existing database. Therefore, discarding large parts of sequence data.
                    </li>
                    <li>Ignores novel players in the microbiome.
    
                    </li>
                </ul>
            </div>
            <div class="paragraph comparison-item seaweed-text">
                <h4>Robust Correlation with Diseases</h4>
                <ul>
                    <li>Generate accurate outlook on roles of specific strains of bacteria in health and diseases
                        without
                        being biased by taxonomic relationships between bacteria.</li>
                </ul>
                </div>

                <div class="paragraph comparison-item gray-text">
                <h4>Spurious Correlation with Diseases
                </h4>
                <ul>
                    <li>Because of biased introduced from using taxonomy to analyze microbiome data, it cannot generate
                        accurate outlook on roles of specific strains of bacteria in health and diseases.
                    </li>
                </ul>
                </div>

                <div class="paragraph comparison-item seaweed-text">
                <h4>Natural Consortium Based on the Foundation Guild</h4>
                <ul>
                    <li>Bacteria in the LBP consortium belongs to the same Foundation Guild, therefore, they are
                        naturally
                        compatible with each other and are competitive against pathogens.</li>
                </ul>
            </div>


                <div class="paragraph comparison-item gray-text">
                <h4>Artificial Consortium or Single Strains
                </h4>
                <ul>
                    <li>Because the bacteria in the consortium are arbitrarily selected, thus, they are not compatible
                        with
                        each other.
                    </li>
                    <li>
                        A single strain of bacteria as treatment cannot compete in the complex gut ecosystem.
                    </li>
                </ul>
                </div>
        </div>

        <!--
        <div class="square space-cadet">
            <ul class="bullet">
                <li>Lower HbA1c (sugar) levels</li>
                <li>Glycemic levels under control</li>
                <li>Lost 10-20% of their body weight</li>
                <li>Alleviate childhood obesity</li>
                <li>Longer healthspan</li>
            </ul>
        </div>
        -->

    </section>

    <section class="refrences">
        <h2>Refrences</h2>

        <ol>
            <li>Zhao, L., et al., Gut bacteria selectively promoted by dietary fibers alleviate type 2 diabetes.
                Science, 2018. 359(6380): p. 1151-1156.
            </li>
            <li>Costea, P.I., et al., Towards standards for human fecal sample processing in metagenomic studies. Nature
                biotechnology, 2017. 35(11): p. 1069.
            </li>
            <li>Zhang, X., et al., Modulation of gut microbiota by berberine and metformin during the treatment of
                high-fat diet-induced obesity in rats. Scientific reports, 2015. 5: p. 14405.
            </li>
            <li>Xu, J., et al., Structural modulation of gut microbiota during alleviation of type 2 diabetes with a
                Chinese herbal formula. The ISME journal, 2015. 9(3): p. 552-562.
            </li>
            <li>Wang, J., et al., Modulation of gut microbiota during probiotic-mediated attenuation of metabolic
                syndrome in high fat diet-fed mice. The ISME journal, 2015. 9(1): p. 1.
            </li>
            <li>Xiao, S., et al., A gut microbiota-targeted dietary intervention for amelioration of chronic
                inflammation underlying metabolic syndrome. FEMS microbiology ecology, 2014. 87(2): p. 357-367.
            </li>
            <li>Zhao, L., The gut microbiota and obesity: from correlation to causality. Nature Reviews Microbiology,
                2013. 11(9): p. 639-647.
            </li>
            <li>Zhang, C., et al., Structural modulation of gut microbiota in life-long calorie-restricted mice. Nature
                communications, 2013. 4(1): p. 1-10.
            </li>
            <li>Shen, J., M.S. Obin, and L. Zhao, The gut microbiota, obesity and insulin resistance. Molecular aspects
                of medicine, 2013. 34(1): p. 39-58.
            </li>
            <li>Fei, N. and L. Zhao, An opportunistic pathogen isolated from the gut of an obese human causes obesity in
                germfree mice. The ISME journal, 2013. 7(4): p. 880-884.
            </li>
            <li>Zhang, X., et al., Structural changes of gut microbiota during berberine-mediated prevention of obesity
                and insulin resistance in high-fat diet-fed rats. PloS one, 2012. 7(8).
            </li>
            <li>Zhang, C., et al., Structural resilience of the gut microbiota in adult mice under high-fat dietary
                perturbations. The ISME journal, 2012. 6(10): p. 1848-1857.
            </li>
            <li>Wang, T., et al., Structural segregation of gut microbiota between colorectal cancer patients and
                healthy volunteers. The ISME journal, 2012. 6(2): p. 320-329.
            </li>
            <li>
                Zhang, C., et al., Interactions between gut microbiota, host genetics and diet relevant to development
                of metabolic syndromes in mice. The ISME journal, 2010. 4(2): p. 232-241.

            </li>
            <li>Li, M., et al., Symbiotic gut microbes modulate human metabolic phenotypes. Proceedings of the National
                Academy of Sciences, 2008. 105(6): p. 2117-2122.
            </li>
        </ol>
    </section>

    <div id="contact-us-modal" class="modal">
        <!-- Modal content -->
        <div class="contact-modal-content">
            <form class="contact" action="https://submit-form.com/kguz4w2C" target="_self">
                <span class="close">&times;</span>
                <h1>Contact Us</h1>
                <label for="name">Name</label><br>
                <input type="text" id="name" name="name"><br>
                <label for="email">Email</label><br>
                <input type="email" id="email" name="email">
                <label for="phone">Phone</label><br>
                <input type="number" id="phone" name="phone">

                <label for="message">Message</label><br>
                <textarea type="text" id="message" name="message"></textarea>

                <form action="?" method="POST">
                    <div class="g-recaptcha" data-sitekey="6LeHPDYaAAAAAI_xDWPLq2cwYnbhD6i-69OWOSuw
                        "></div>
                    <br />
                    <input type="submit" value="Submit">
                </form>
                <input type="submit">
            </form>
        </div>
    </div>

    <footer>
        Copyright &copy; 2021 Notitia Biotechnologies. All Rights Reserved.
    </footer>
    <script src="https://code.jquery.com/jquery-3.5.1.slim.min.js"
        integrity="sha256-4+XzXVhsDmqanXGHaHvgh1gMQKX40OUvDEBTu8JcmNs=" crossorigin="anonymous"></script>
    <script src="js/main.js"></script>
    <script src="https://unpkg.com/aos@next/dist/aos.js"></script>
    <script>
        AOS.init({
            //anchorPlacement: "bottom-center",
            offset: 200,
            duration: 1000,
        });
    </script>
</main>

</html>