<!DOCTYPE html>
<html lang="en">

<head>
    <meta charset="utf-8">

    <title>Notitia | Our Science</title>
    <meta name="description" content="Homepage for NotitiaBio">
    <meta name="author" content="Notitia">
    <meta name="viewport" content="width=device-width, initial-scale=1">


    <link rel="stylesheet" href="style.css">
    <link rel="icon" href="images/small-logo.png" type="image/png">

    <link rel="stylesheet" href="https://unpkg.com/aos@next/dist/aos.css" />
</head>
<main>
    <div class="hamburger">
        <div></div>
        <div></div>
        <div></div>
    </div>
    <nav class="top-nav">
        <img src="images/Original.svg" alt="notitia logo">
        <a href="index.html">Home</a>
        <a href="about.html">About Us</a>
        <div class="dropdown" id="hide-mobile">Our Science
            <div class="dropdown-content">
                <a href="pillars.html">The Four Pillars</a>
                <a href="platform.html">Our Platform</a>
            </div>
        </div>
        <a class="show-mobile" href="pillars.html">The Four Pillars</a>
        <a class="show-mobile" href="platform.html">Our Platform</a> 
        <a href="general-public.html">General Public</a>
        <a href="patients.html">Patients</a>
        <a href="investors.html">Investors & News</a>
    </nav>
    <nav class="secondary-nav">
        <a href="pillars.html">The Four Pillars</a>
        <a href="technologies.html">Our Technologies</a>
        <a href="comparison.html" class="active">Comparison</a>
    </nav>
    <button class="contact-us" id="contact-us">Contact Us</button>
    <section class="hero science-hero">
        <div class="text-container">
            <h1>Our Science</h1>
        </div>
    </section>

    <section class="frame grid banner-2" data-aos="shrink">
        <h1>The Four Pillars of Understanding Microbiome</h1>
        <p>The complex and dynamic nature of gut microbiota raises significant challenges for understanding its role in human health and diseases. To overcome these challenges, our scientific co-founder, Dr. Liping Zhao, has spent the past three decades to develop a conceptual framework encompassing Four Pillars to guide microbiome research and promote understanding of the role of gut microbiome in human health and diseases.</p>
    </section>

    <section class="frame " data-aos="shrink">
        <div>
            <h1>Pillar 1: We must move from studying association to studying causation.</h1>
            <div class="chicken-egg">
                <img src="images/egg.png" alt="egg">
                <div class="chicken-arrow">

                    <img src="images/right.png" alt="arrow">
                    <img src="images/right.png" alt="arrow">
                </div>
                <img src="images/hen.png" alt="chicken">
        </div>
            <p>
                To date, plenty of studies have shown that patients of various diseases have different microbiota compared to healthy individuals. There is also significant evidence identifying diet as the main force for shaping the composition of the gut microbiota. Thus, the key to changing the gut microbiota lies in dietary changes. However, in many cases it is still unclear whether changes in a patient’s gut microbiota are the cause or the end result of a disease? Changing one’s diet can simultaneously impact gut microbiota composition and patient health outcome. However, does the changed gut microbiota also improve health? Or is it merely an end result of the improved health?
                <br><br>
                Some bacteria are more frequently found in healthy people, while other bacteria are more frequently found in patients. Nevertheless, such association evidence cannot prove whether a bacterium is beneficial or harmful. Moving from studying association to studying causality, we must isolate the gut bacteria and inoculate them into germ-free animal models to test their health effects. In 2012, Dr. Zhao found a particular pathogen overgrown in a morbidly obese patient's gut. After receiving a microbiome-targeted diet, the pathogen quickly reduced from 1/3 of the patient’s gut bacteria population to a non-detectable level. At the same time, this patient lost 110 lbs of his initial body weight and recovered from type 2 diabetes. Dr. Zhao hypothesized that this pathogen played a causal role in the patient's obesity and type 2 diabetes. This pathogen was then isolated from the patient's baseline samples and transferred to a germ-free mice gut. <strong>The lab results clearly showed that this pathogen induced obesity, fatty liver, and early diabetes in the recipient mice.</strong> Dr. Zhao then identified the exact molecule from the pathogenic bacteria that is responsible for causing the diseases. For the first time, science proved that certain “bad” gut bacteria can CAUSE obesity and type 2 diabetes.
                <img src="" alt="weight loss">
                <br><br>
                In 2018, Dr. Zhao found that some bacteria are scarce in the gut of type 2 diabetes patients. After patients received a microbiome-targeted diet, these bacteria became abundant in the gut. Meanwhile, patients showed significant improvement in their health indicators. He then isolated thesebacteria and transferred them to diabetic mice. Amazingly, the bacteria helped mice recover from their diabetes. These bacteria are proven beneficial!                <br><br>
                We believe that conducting causality studies with a molecular level of rigor is key to establish our confidence to use bacteria and their effector molecules as biomarkers and targets for disease prevention or treatment.
            </p>
        </div>
    </section>

    <section class="frame step-1" data-aos="shrink">
        <div>
            <h1>Pillar 2: We must identify “good” and “bad” bacteria at the strain level. </h1>
            <p>
                Strains are the basic units of bacteria just as human individuals are the basic units of the human species.
                <br><br>
                As a species, human individuals share 99.9% of their genetic code which means that they are biologically similar to each other. However, bacterial strains in the same species only share 70% of their genetic code. As a reference, humans and mice still share 90% of their genetic code. This means that the difference between two strains within the same bacterial species can be greater than the difference between humans and mice.
                <br><br>
                Because of such vast genetic differences within the same bacterial species, some bacteria can be pathogenic and others can be beneficial. We cannot collectively talk about bacterial species as good or bad. In 2016, Dr. Zhao found one strain that can aggravate gut inflammation and colitis in a supposedly beneficial species of gut bacteria. In 2017, he discovered that a healthy diet does not promote all the strains belonging to a supposedly beneficial species. Many of the strains were, in fact, non-responders to the healthy diet.
                <br><br>
                We cannot use species as functional units to understand the role of the gut microbiota in human health and diseases because their members often do not behave in the same way. We need to reach strain-level specificity if we desire to identify whether a bacterium is good or bad to our health.
            </p>
        </div>
    </section>

    <section class="frame step-2" data-aos="shrink">
        <div>
            <h1>Pillar 3: We must remember that bacteria form ecological guilds.</h1>
            <p>
                Members in an ecosystem (e.g., a rain forest) are dependent on each other. They form an ecological structure called a guild. Each guild is a coherent functional group of various members which either thrive or decline together. Dr. Zhao found that bacterial strains in the gut ecosystem also organize themselves into guilds. In 2015, Dr. Zhao identified three beneficial and nine harmful guilds in the gut of genetically obese children. This was the first time that the guild concept was utilized in human microbiome research
                <br><br>
                In nature, we understand that some members within an ecosystem are more important than others. The tall tree species in the rain forest are known as the foundation species because they dominate the ecosystem in abundance and influence, structuring and stabilizing the entire rainforest. Foundation species are different from keystone species, which have limited influence. If you lose a keystone species, you may lose part of the ecosystem. If you lose the foundation species, you will eventually lose everything. Are there any gut bacteria that also work as the foundation species to structure and stabilize the healthy gut microbiota?
                <br><br>
                In 2018, Dr. Zhao discovered the first example of the foundation species in the gut microbiota. When at a high level, this guild of beneficial bacteria can effectively change the entire gut environment and make it unfavorable to pathogens in the gut. This guild plays a fundamental role in structuring and stabilizing the healthy gut microbiota. Bacteria from various groups can be found within this guild. We have named this group of bacteria the "Foundation Guild&trade;" of the gut microbiome.
                <br><br>
                Because of the critical role that the Foundation Guild&trade; plays in a healthy gut microbiota, promoting and restoring this group should be the focus of any effective microbiome therapeutics.
            </p>
        </div>
    </section>

    <section class="frame step-3" data-aos="shrink">
        <div>
            <h1>Pillar 4: We must not rely on existing reference databases for microbiome research. </h1>
            <p>
                Our current knowledge of the gut microbiome is still limited. If we relied on existing reference databases to analyze microbiome data, we would limit research to what is already known, re-discover what others have previously found, and potentially over look novel bacteria that are important to human health.
                <br><br>
                We need to adopt a system-level, non-targeted, discovery approach for understanding gut microbiota in human health and diseases. We need to develop a reference-independent strategy for analyzing microbiome data. In the potentially beneficial guilds identified by Dr. Zhao, only about 20% can be characterized. The other 80% represent novel bacteria that have never been discovered before.
            </p>
        </div>
    </section>

    <section class="frame" data-aos="shrink">
        <div>
            <h1>Notitia's Foundation Guild&trade; Microbiome R&D Platform</h1>
            <p>
                Based on these Four Pillars and the ground-breaking scientific discoveries, we have developed the Foundation Guild&trade; Microbiome R&D Platform that serves as the basis of various technologies for promoting the Foundation Guild&trade; in patients and the generally healthy population.
                <br><br>
                Notitia's platform and technologies are centered on the Foundation Guild&trade; gut bacteria, which has been proven to have a causal relationship with human health and diseases. Our proprietary data analysis pipeline allows us to study members of the Foundation Guild&trade; at the strain-level and examine relationships within their respective guilds. This entire process does not rely on existing references, enabling us to discover new bacteria that could be important to human health.
                <br><br>
                One of the most important aspects of our platform is its ability to analyze and adjust Foundation Guild&trade; gut bacteria in each individual. The platform is made up of three unique technologies:                
            </p>
            <ol>
                <li>Testing the Foundation Guild&trade;
                    <br><br>
                    Notitia has created a predictive analysis method to identify the Foundation Guild&trade; in each person's gut microbiota. We can analyze an individual's fecal sample and investigate: does this individual have Foundation GuildTM bacteria in their gut? If so, what are the specific members of their Foundation GuildTM? What kind of nutrients are required to support their growth? The analysis can also predict the optimal state of the Foundation GuildTM for each person and keep track of their recovery and progression.
                    <br><br>
                    (Example graphics) In the example on the left, the analysis predicted the optimal healthy state of the Foundation Guild&trade; of the individual. After being on the microbiome nutrition, the same test showed that the individual's gut microbiome moved closer to its optimal state, indicating that the Foundation Guild&trade; is recovering.
                    <br><br>
                </li>
                <li>Growing the Foundation Guild&trade;
                    <br><br>
                    Notitia has invented a system to isolate and grow the Foundation Guild&trade; bacteria from fecal samples. We can exclude all other microorganisms inside the fecal sample and isolate only the Foundation Guil&trade; bacteria. Regardless of the number of specific bacteria one person has in their Foundation Guild&trade;, our method allows us to isolate the entire group as a whole and store it in a cryogenic repository for future use.
                    <br><br>
                    Additionally, this technology enables us to create LBP (live biotherapeutic products) drugs, which contain the Foundation Guil&trade; bacteria to administer to patients that need microbiota transplantation.
                    <br><br>
                </li>
                <li>Nutrition for the Foundation Guild&trade;
                    <br><br>
                    Notitia has designed formulas that contain the proper nutrients to support the Foundation Guild&trade; growth for patients and the generally healthy. These formulas contain critical ingredients such as complex carbohydrates and phytochemicals with antioxidant activity. Each ingredient is designed to be efficiently utilized by the Foundation Guild&trade;.
                    <br><br>
                    The clinical formula focuses on supporting as many types of Foundation Guil&trade; bacteria there are in the human gut. In contrast, the non-clinical formula aids the most common Foundation Guild&trade; members and is a versatile food ingredient.  
                </li>
            </ol>
        </div>

    </section>


        <div class="comparison">
            <div class="comparison-header">
                <img src="images/notitia.png">
            </div>
            <h1 class="comparison-vs">VS.</h1>
            <h1 class="comparison-header red-text">Competitor</h1>
            <div class="paragraph comparison-item seaweed-text">
                <h4>Guild-based Analysis</h4>
                <ul>
                    <li>Does not rely on existing database.</li>
                    <li>Able to discover novel bacteria in the microbiome.
                    </li>
                </ul>
                </div>
                
                <div class="paragraph comparison-item red-text">
                <h4>Taxon-based Analysis
                </h4>
                <ul>
                    <li>Relies on existing database. Therefore, discarding large parts of sequence data.
                    </li>
                    <li>Ignores novel players in the microbiome.
    
                    </li>
                </ul>
            </div>
            <div class="paragraph comparison-item seaweed-text">
                <h4>Robust Correlation with Diseases</h4>
                <ul>
                    <li>Generate accurate outlook on roles of specific strains of bacteria in health and diseases
                        without
                        being biased by taxonomic relationships between bacteria.</li>
                </ul>
                </div>

                <div class="paragraph comparison-item red-text">
                <h4>Spurious Correlation with Diseases
                </h4>
                <ul>
                    <li>Because of biased introduced from using taxonomy to analyze microbiome data, it cannot generate
                        accurate outlook on roles of specific strains of bacteria in health and diseases.
                    </li>
                </ul>
                </div>

                <div class="paragraph comparison-item seaweed-text">
                <h4>Natural Consortium Based on the Foundation Guild&trade;</h4>
                <ul>
                    <li>Bacteria in the LBP consortium belongs to the same Foundation Guild&trade;, therefore, they are
                        naturally
                        compatible with each other and are competitive against pathogens.</li>
                </ul>
            </div>


                <div class="paragraph comparison-item red-text">
                <h4>Artificial Consortium or Single Strains
                </h4>
                <ul>
                    <li>Because the bacteria in the consortium are arbitrarily selected, thus, they are not compatible
                        with
                        each other.
                    </li>
                    <li>
                        A single strain of bacteria as treatment cannot compete in the complex gut ecosystem.
                    </li>
                </ul>
                </div>
        </div>

    </section>

    <section class="refrences">
        <h2>Refrences</h2>

        <ol>
            <li>Zhao, L., et al., Gut bacteria selectively promoted by dietary fibers alleviate type 2 diabetes.
                Science, 2018. 359(6380): p. 1151-1156.
            </li>
            <li>Costea, P.I., et al., Towards standards for human fecal sample processing in metagenomic studies. Nature
                biotechnology, 2017. 35(11): p. 1069.
            </li>
            <li>Zhang, X., et al., Modulation of gut microbiota by berberine and metformin during the treatment of
                high-fat diet-induced obesity in rats. Scientific reports, 2015. 5: p. 14405.
            </li>
            <li>Xu, J., et al., Structural modulation of gut microbiota during alleviation of type 2 diabetes with a
                Chinese herbal formula. The ISME journal, 2015. 9(3): p. 552-562.
            </li>
            <li>Wang, J., et al., Modulation of gut microbiota during probiotic-mediated attenuation of metabolic
                syndrome in high fat diet-fed mice. The ISME journal, 2015. 9(1): p. 1.
            </li>
            <li>Xiao, S., et al., A gut microbiota-targeted dietary intervention for amelioration of chronic
                inflammation underlying metabolic syndrome. FEMS microbiology ecology, 2014. 87(2): p. 357-367.
            </li>
            <li>Zhao, L., The gut microbiota and obesity: from correlation to causality. Nature Reviews Microbiology,
                2013. 11(9): p. 639-647.
            </li>
            <li>Zhang, C., et al., Structural modulation of gut microbiota in life-long calorie-restricted mice. Nature
                communications, 2013. 4(1): p. 1-10.
            </li>
            <li>Shen, J., M.S. Obin, and L. Zhao, The gut microbiota, obesity and insulin resistance. Molecular aspects
                of medicine, 2013. 34(1): p. 39-58.
            </li>
            <li>Fei, N. and L. Zhao, An opportunistic pathogen isolated from the gut of an obese human causes obesity in
                germfree mice. The ISME journal, 2013. 7(4): p. 880-884.
            </li>
            <li>Zhang, X., et al., Structural changes of gut microbiota during berberine-mediated prevention of obesity
                and insulin resistance in high-fat diet-fed rats. PloS one, 2012. 7(8).
            </li>
            <li>Zhang, C., et al., Structural resilience of the gut microbiota in adult mice under high-fat dietary
                perturbations. The ISME journal, 2012. 6(10): p. 1848-1857.
            </li>
            <li>Wang, T., et al., Structural segregation of gut microbiota between colorectal cancer patients and
                healthy volunteers. The ISME journal, 2012. 6(2): p. 320-329.
            </li>
            <li>
                Zhang, C., et al., Interactions between gut microbiota, host genetics and diet relevant to development
                of metabolic syndromes in mice. The ISME journal, 2010. 4(2): p. 232-241.

            </li>
            <li>Li, M., et al., Symbiotic gut microbes modulate human metabolic phenotypes. Proceedings of the National
                Academy of Sciences, 2008. 105(6): p. 2117-2122.
            </li>
        </ol>
    </section>

    <div id="contact-us-modal" class="modal">
        <div class="contact-modal-content">
            <form class="contact" action="https://formspree.io/f/xqkgjjzv" method="POST">
                <span class="close">&times;</span>
                <h1>Contact Us</h1>
                <label for="name">Name</label><br>
                <input type="text" id="name" name="name"><br>
                <label for="email">Email</label><br>
                <input type="email" id="email" name="email">
                <label for="phone">Phone</label><br>
                <input type="number" id="phone" name="phone">

                <label for="message">Message</label><br>
                <textarea type="text" id="message" name="message"></textarea>

                <input type="submit">
            </form>
        </div>
    </div>

    <footer>
        Copyright &copy; 2021 Notitia Biotechnologies. All Rights Reserved.
    </footer>
    <script src="https://code.jquery.com/jquery-3.5.1.slim.min.js"
        integrity="sha256-4+XzXVhsDmqanXGHaHvgh1gMQKX40OUvDEBTu8JcmNs=" crossorigin="anonymous"></script>
    <script src="js/main.js"></script>
    <script src="https://unpkg.com/aos@next/dist/aos.js"></script>
    <script>
        AOS.init({
            //anchorPlacement: "bottom-center",
            offset: 200,
            duration: 1000,
        });
    </script>
</main>

</html>